### ISSUE TITLE
Support for special characters in FlowRouter

### ISSUE THREAD

**Issue by TimoRuetten** [kadirahq/flow-router#599]
We try to add a special character to our app while using FlowRouter - but it seems to be a problem.

Like GoogleMaps its doing we want to add a @-sign in our ULR when having directly LatLng data in it. Example:

app.io/location/@Lat,Lng

Sadly the FlowRouter is always encoding our @ to %40 in our url which looks very ugly. Is there a way to prevent this encoding ?

**Comment by sahanDissanayake** [comment]
Same issue here

Tried to set the following but the url comes out encoded

```
FlowRouter.setQueryParams({view: 'そししきききちち'})
```

Maybe the issue is here @arunoda  ? @TimoRuetten 
https://github.com/kadirahq/flow-router/blob/56c5e7e8d025affd41d8532524470fe9737fa189/client/router.js#L164

**Comment by TimoRuetten** [comment]
@sahanDissanayake Yes thats the. The params or query object will have the values not encoded but the url params and queries will be always encoded. It would be nice to prevent this.

**Comment by TimoRuetten** [comment]
Hey @arunoda - is there any way that you will fix this or do we have to do a custom solution for this ?

**Comment by TimoRuetten** [comment]
It seems that the problem is not the FlowRouter - its the path-to-regex instead. When using the SSR router the params will not be encoded by kadira if I see it correctly. The problem is the compile() method by path-to-regex package. This is relay sad

**Comment by nicooprat** [comment]
Ended at the same conclusion as @TimoRuetten: https://github.com/pillarjs/path-to-regexp/issues/42.

Any news or workaround for this? I just need to have commas in URLs (eg. for GPS coordinates).
